integration-tests/utils/integration-helpers.js

```diff
@@ -26,22 +26,29 @@ const getTestWorkspaceName = () => `test-workspace-${Math.floor(Math.random() *
 
 const makeWorkspace = withSignedInPage(async ({ page, billingProject }) => {
   const workspaceName = getTestWorkspaceName()
-  await page.evaluate((name, billingProject) => {
-    return window.Ajax().Workspaces.create({ namespace: billingProject, name, attributes: {} })
-  }, workspaceName, billingProject)
-
-  rawConsole.info(`created workspace: ${workspaceName}`)
+  try {
+    await page.evaluate((name, billingProject) => {
+      return window.Ajax().Workspaces.create({ namespace: billingProject, name, attributes: {} })
+    }, workspaceName, billingProject)
 
+    rawConsole.info(`Created workspace: ${workspaceName}`)
+  } catch (e) {
+    throw Error(`Failed to create workspace: ${workspaceName} with billing project ${billingProject}`)
```

Comment:
I think these error messages will be much clearer when they fail - thanks!

Reply:
I saw this test fail 2/100 times because it failed to create a workspace. Is this something that would get covered by the retries you mentioned? If not I can make a separate ticket for this.

Reply:
Yes, but I think retry is out of scope for this ticket. I think the improvements here will reduce a lot of failure noise / flakiness

Reply:
I wonder if it makes sense to investigate why workspace creation fails before adding a retry, to make sure retry wouldn't conceal a back-end issue.